### PR TITLE
Add prevalidation jobs to ironic-image for OCP 4.10

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-release-4.10__prevalidation.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-metal-ipi-prevalidation
+  run_if_changed: .*packages-list.*
+  steps:
+    cluster_profile: packet
```
Comment (andfasano) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-release-4.10__prevalidation.yaml:54]: must be `equinix-ocp-metal`

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-release-4.10__prevalidation.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    workflow: baremetalds-e2e-ipv4
+- as: e2e-metal-ipi-virtualmedia-prevalidation
+  run_if_changed: .*packages-list.*
+  steps:
+    cluster_profile: packet
```
Comment (andfasano) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-release-4.10__prevalidation.yaml:59]: And this one

**Review by iurygregory — APPROVED** [review]

**Pull request merged**